.github/workflows/ci.yml

```diff
@@ -17,10 +17,10 @@ jobs:
       run: |
         git submodule init && git submodule update --recursive --remote
 
-    - name: Set up Python 3.9
+    - name: Set up Python 3.11
```

Comment:
Hmm, I'm actually a bit surprised that the tests were running correctly before for case matching (e.g. in thens/alignment.py)

Reply:
Agree.  Pattern matching didn't land until python 3.10.

Reply:
```
[Unit Test with pytest/build-linux]   ✅  Success - Main Lint with flake8
[Unit Test with pytest/build-linux] ⭐ Run Main Test with pytest
[Unit Test with pytest/build-linux]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/6] user= workdir=
| ============================= test session starts ==============================
| platform linux -- Python 3.10.12, pytest-8.1.1, pluggy-1.4.0 -- /usr/bin/python3
| cachedir: .pytest_cache
| rootdir: /home/scott/projects/ifc-gherkin-rules
| configfile: pyproject.toml
collected 239 items         
```

looking at the logs the github action appears to use system python (`/usr/bin/python3`) of the docker container, which happens to be 3.10 - which is why the tests were passing before even with the yaml file specifying 3.9.